src/common/main.lua

```diff
@@ -88,40 +88,92 @@ function ant.init()
 
 	createActionHandler(TokenSpecActionMap.Transfer, function(msg)
 		local recipient = msg.Tags.Recipient
-		utils.validateOwner(msg.From)
+		if msg.From ~= Owner then
+			if msg.reply then
+				msg.reply({
+					Action = "Transfer-Error",
+					["Message-Id"] = msg.Id,
+					Error = "Insufficient Balance!",
+				})
+			else
+				ao.send({
+					Target = msg.From,
+					Action = "Transfer-Error",
+					["Message-Id"] = msg.Id,
+					Error = "Insufficient Balance!",
+				})
```

Comment:
nit - wrap these in a Send util like so https://github.com/ar-io/ar-io-network-process/blob/develop/src/main.lua#L128-L135

Reply:
makes sense https://github.com/ar-io/ar-io-ant-process/pull/59/commits/79bd00130f8891521da4dfa74b5c0e2c9870fd21